src/parser/components.rs

```diff
@@ -131,15 +131,27 @@ impl From<Component<'_>> for Other {
 
 impl From<Component<'_>> for InnerComponent {
     fn from(component: Component) -> Self {
-        Self {
-            properties: component
-                .properties
+        fn is_multi(property: &Property) -> bool {
+            property.name.to_string() == "ATTENDEE"
```

Comment:
seems it's only ever attendee properties right?

Reply:
Yeah I agree. I didn't like it so much myself but was debating on exactly where to put it. Also, yes, there are many more properties to be added but I had started with Attendee as a proof of concept. I will update this.